packages/client-isomorphic-fetch/README.md

```diff
@@ -1,3 +1,3 @@
-# @iroha2/client-isomorphic-fetch
+# `@iroha2/client-isomorphic-fetch`
 
-In Node.js fallbacks to `node-fetch` package, otherwise to native `fetch()`.
+In Node.js, `@iroha2/client-isomorphic-fetch` fallbacks to either `node-fetch` package or native `fetch()`.
```

Comment:
I would say that it uses `node-fetch` in Node.js environment, i.e. in CJS mode, and uses native Fetch API in Browser, i.e. in ESM mode.

Reply:
like this?

```suggestion
`@iroha2/client-isomorphic-fetch` uses `node-fetch` package in CJS mode (in Node.js) and native Fetch API in ESM mode (in a browser).
```